src/EnergyPlus/DaylightingDevices.cc

```diff
@@ -205,6 +205,7 @@ namespace DaylightingDevices {
 
     void clear_state() {
         COSAngle.clear();
+        COSAngle.allocate(NumOfAngles);
```

Comment:
Needed to reallocate since it was allocated to a specific amount in the declaration.